content/en/blog/_posts/2024-04-17-kubernetes-1.30.md

```diff
@@ -0,0 +1,218 @@
+cycle and the vibrant support from our community.
+
+This release consists of 45 enhancements. Of those enhancements, 17 have graduated to Stable, 18 are
+entering Beta, and 10 have graduated to Alpha.
+
+## Release theme and logo
+
+(logo placeholder-pending high res file and resize)
+
+Kubernetes v1.30 makes your clusters cuter!
+
+Kubernetes is built and released by thousands of people from all over the world and all walks of
+life. Most contributors are not being paid to do this; we build it for fun, to solve a problem, to
+learn something, or for the simple love of the community. Many of us found our homes, our friends,
+and our careers here. The Release Team is honored to be a part of the continued growth of
+Kubernetes. 
+
+For the people who built it, for the people who release it, and for the furries who keep all of our
+clusters online, we present to you Kubernetes v1.30: Uwubernetes, the cutest release to date. The
+name is a portmanteau of “kubernetes” and “UwU,” an emoticon used to indicate happiness or cuteness.
+We’ve found joy here, but we’ve also brought joy from our outside lives that helps to make this
+community as weird and wonderful and welcoming as it is. We’re so happy to share our work with you. 
+
+UwU :heart:
+
+## Improvements that graduated to stable in Kubernetes v1.30 {#graduations-to-stable}
+
+_This is a selection of some of the improvements that are now stable following the v1.30 release._
+
+### Robust VolumeManager reconstruction after kubelet restart ([SIG Storage](https://github.com/kubernetes/community/tree/master/sig-storage))
+
+This is a volume manager refactoring that allows the kubelet to populate additional information about how existing volumes are mounted during the kubelet startup. In general, this makes volume cleanup after kubelet restart or machine reboot more robust.
+
+This does not bring any changes for user or cluster administrators. We used the feature process and feature gate `NewVolumeManagerReconstruction` to be able to fall back to the previous behavior in case something goes wrong. Now that the feature is stable, the feature gate is locked and cannot be disabled.
+
+### Prevent unauthorized volume mode conversion during volume restore ([SIG Storage](https://github.com/kubernetes/community/tree/master/sig-storage))
+
+With the release of Kubernetes 1.30, the ability to inherently prevent unauthorized users from converting the volume mode of a PVC, when it is created from an existing VolumeSnapshot, moves to GA. Cluster administrators will now have to specifically grant permissions in Kubernetes to authorized users, like storage backup vendors, to allow them to perform such actions. 
+For more information, see [converting the volume mode of a Snapshot](/docs/concepts/storage/volume-snapshots/#convert-volume-mode).
+
+### Pod Scheduling Readiness ([SIG Scheduling](https://github.com/kubernetes/community/tree/master/sig-scheduling))
+
+Blurb for 3521 goes here.
```

Comment:
```suggestion
Pod Scheduling Readiness introduces a new phase at the start of a Pod's lifecycle. A Pod won't be scheduled until its `schedulerGates` are all cleared. This feature empowers external integrators to intercede during the pre-scheduling cycle of a Pod, thereby enabling the orchestration of customized workflows that cater to specific in-house requirements.
```

Reply:
Rather than talking about “introducing a new phase”, how about explaining that the Pod lifecycle
has a phase where the scheduler checks if the Pod has scheduling suspended?

Reply:
@natalisucks ptal

Reply:
@sftim I have the understanding that Pods are ready to be scheduled once they're created, and the `schedulingGates` field can only be initialized when the Pod is created. With that being the case, could we possibly restate that first sentence as "This feature enhances the Pod lifecycle by allowing you to control when a Pod can be scheduled, downsizing the churn that the Kubernetes scheduler might encounter."

Reply:
> @sftim I have the understanding that Pods are ready to be scheduled once they're created

Sometimes. Pods are ready to be scheduled if either: they are created without scheduling gates, or if they were created with scheduling gates that have since been removed.

---

I'm not particularly keen on “this feature&hellip;” or “enhances” for _graduated_ features. At graduation, we're describing the stable behavior of Kubernetes. We're also discussing something that's been on by default for at least one minor release and possibly many. Yes, it absolutely is an enhancement, but to a typical end user the change is around the level of support, not actually the behavior.

Reply:
@sftim Do you have a suggestion to replace "feature" as the descriptor of this enhancement as opposed to repeating the name of it? Or would you be more keen on repeating the name of the enhancement?